Projects/Playground/Playground.Forms.UI/Pages/ParentContentPage.xaml

```diff
@@ -8,9 +8,11 @@
              x:Name="ParentPage">
     <ContentPage.Content>
         <StackLayout>
-            <views:ChildContentView ViewModel="{Binding ChildViewModel1}" IsEnabled="{Binding ViewModel.ChildViewModelEnabled, Source={x:Reference ParentPage}}"/>
-            <Button Text="Change Child Text" Command="{Binding ChangeButtonCmd}" />
-            <Button Text="Toggle Child Enabled" Command="{Binding ToggleChildEnabledCmd}" />
+            <views:ChildContentView ViewModel="{Binding Source={x:Reference ParentPage}, Path=ViewModel.ChildViewModel1}" IsEnabled="{Binding ViewModel.ChildViewModelEnabled, Source={x:Reference ParentPage}}"/>
```

Comment:
Is there a reason why "Source" needs to be specified for either of these databinding expressions?

Reply:
Not in this specific context, but if Source is not specified then the binding works a grand total of one time, after which the child view's base.BindingContext is no longer the same as the parent view's ViewModel and it wouldn't be able to find the property ChildViewModel1 in the class 'ChildViewModel'.